packages/auth/demo/README.md

```diff
@@ -74,3 +89,43 @@ The demo page by default runs against the actual Auth Backend. To run against th
 yarn run demo:emulator
 ```
 
+## Troubleshooting
+
+### Errors about dependency not being installed, example `lerna: command not found`
+  
+  Ensure that you run `yarn` to install dependencies.
+
```

Comment:
I also run into a similar problem when run `firebase use --add`.
We can also add another troubleshooting here -
`Failed to list Firebase projects.`
Logout, re-login
firebase logout && firebase login